packages/suite/src/actions/wallet/send/sendFormThunks.ts

```diff
@@ -0,0 +1,596 @@
+import BigNumber from 'bignumber.js';
+import { G, A } from '@mobily/ts-belt';
+
+import { createThunk } from '@suite-common/redux-utils';
+import {
+    Account,
+    ComposeActionContext,
+    FormState,
+    PrecomposedTransactionFinal,
+    PrecomposedTransactionFinalCardano,
+} from '@suite-common/wallet-types';
+import { MetadataAddPayload } from '@suite-common/metadata-types';
+import { notificationsActions } from '@suite-common/toast-notifications';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import {
+    selectAccounts,
+    selectDevice,
+    replaceTransactionThunk,
+    addFakePendingCardanoTxThunk,
+    accountsActions,
+    addFakePendingTxThunk,
+    syncAccountsWithBlockchainThunk,
+    selectSpecificTokenDefinition,
+} from '@suite-common/wallet-core';
+import {
+    hasNetworkFeatures,
+    amountToSatoshi,
+    formatAmount,
+    getAccountDecimals,
+    getAreSatoshisUsed,
+    formatNetworkAmount,
+    getPendingAccount,
+    isCardanoTx,
+} from '@suite-common/wallet-utils';
+import TrezorConnect, { SignedTransaction } from '@trezor/connect';
+import { cloneObject, getSynchronize } from '@trezor/utils';
+
+import { selectRoute } from 'src/reducers/suite/routerReducer';
+import * as modalActions from 'src/actions/suite/modalActions';
+import * as metadataLabelingActions from 'src/actions/suite/metadataLabelingActions';
+import {
+    selectSelectedAccountKey,
+    selectIsSelectedAccountLoaded,
+    selectSelectedAccount,
+} from 'src/reducers/wallet/selectedAccountReducer';
+
+import {
+    selectSendFormDrafts,
+    selectSendSignedTx,
+    selectSendPrecomposedTx,
+    selectPrecomposedSendForm,
+} from 'src/reducers/wallet/sendFormReducer';
+import {
+    selectAreSatsAmountUnit,
+    selectBitcoinAmountUnit,
+} from 'src/reducers/wallet/settingsReducer';
+
+import {
+    signBitcoinTransactionThunk,
+    composeBitcoinTransactionThunk,
+} from './sendFormBitcoinThunks';
+import {
+    signEthereumTransactionThunk,
+    composeEthereumTransactionThunk,
+} from './sendFormEthereumThunks';
+import {
+    signCardanoTransactionThunk,
+    composeCardanoTransactionThunk,
+} from './sendFormCardanoThunks';
+import { signRippleTransactionThunk, composeRippleTransactionThunk } from './sendFormRippleThunks';
+import { signSolanaTransactionThunk, composeSolanaTransactionThunk } from './sendFormSolanaThunks';
+import { MODULE_PREFIX } from './constants';
+import { findLabelsToBeMovedOrDeleted, moveLabelsForRbfAction } from '../moveLabelsForRbfActions';
+import { sendFormActions } from '../sendFormActions';
+
+export const saveFormDraftThunk = createThunk(
```

Comment:
Please change naming of all these thunks. Since they're not used within namespace like before (e. g. `sendForm.<thunk>`), we want to know what kind of functionality they do. For example `dispatch(convertDraftsThunk())` is really weird and nobody will know what it does from the naming itself.

```suggestion
export const saveSendFormDraftThunk = createThunk(
```

Reply:
fixed by c2606f3977